### ISSUE TITLE
Ticket machine-like block

### ISSUE THREAD

**Issue by vedrit** [Railcraft/Railcraft#1566]
A machine built into RailCraft to facilitate the easy creation of tickets.

How I think it might work? A block that will have 3 slots:
1) Paper stack (accept paper from top via hopper/pipes/etc?)
2) Gold ticket (lockable slot by block owner?)
3) Output

There will likely also need to a button to create the ticket.

Maybe additional gold ticket slots can become available when another gold ticket is added? Also, maybe text slots for destination/station/rail line name?

**Comment by liach** [comment]
I'd tweak it into a selling machine. Three types of golden tickets can be put in and can have a price defined. Of course, it will have a secure inventory holding the income and paper for tickets.

**Comment by brisingraerowing** [comment]
I know Computronics has something like this, though IIRC it needs a computer mod (either OpenComputers or ComputerCraft) to actually work.

**Comment by vedrit** [comment]
> I know Computronics has something like this, though IIRC it needs a computer mod (either OpenComputers or ComputerCraft) to actually work.

Yes, OpenComputers has one, but I think it's better that RailCraft has it's own ticket machine.
